.github/workflows/CI.yml

```diff
@@ -99,7 +111,7 @@ jobs:
   verify:
     name: Verify
     needs: [setup, prettier]
-    if: needs.setup.outputs.frontend != 'false' || needs.setup.outputs.workflows != 'false'
+    if: needs.setup.outputs.geometries != 'false' || needs.setup.outputs.workflows != 'false'
```

Comment:
Is this change on purpose - shouldn't it still run on frontend changes? 🤔

Reply:
It now runs on all the web/geo changes and zone + exchange changes. There should be no need to check if the map has updated if we are just changing "pure" frontend components like say the translations.

Reply:
@madsnedergaard merging this now, but if there are any issues, we should spot them pretty easily as we run this and the CirlceCI jobs in parallel.

Reply:
Ah okay, I thought we would need either `.frontend` or `.geometries` to be not false, but makes sense :)

Reply:
Hmm, good point. It might be needed for cypress since it needs verify. I'll monitor the situation and add `.frontend` too if it's needed.